toolchain/lex/tokenized_buffer_test.cpp

```diff
@@ -1215,14 +1215,14 @@ TEST_F(LexerTest, IndentedComments) {
     std::string simd_source =
         source +
         "\"Add a bunch of padding so that SIMD logic shouldn't hit EOF\"";
-    auto& simd_buffer = compile_helper_.GetTokenizedBuffer(source);
+    auto& simd_buffer = compile_helper_.GetTokenizedBuffer(simd_source);
     ASSERT_FALSE(simd_buffer.has_errors());
     EXPECT_THAT(simd_buffer.comments_size(), Eq(1));
   }
 }
 
 TEST_F(LexerTest, MultipleComments) {
-  constexpr llvm::StringLiteral Format = R"(
```

Comment:
Specifically for `StringLiteral`, I think we should fix it to have a `c_str()` method. Would it make sense to disable the tidy until that lands in LLVM and we pull it back in?

I agree with you that this seems like a net negative change.

Reply:
To be sure, I was mainly trying to clarify the edits -- I think it's probably worth keeping on. In particular, the warning in `testing/file_test/file_test_base.cpp` seems like an improvement, and this spot seems more neutral to me. 

Noting the uncertainty of future changes to upstream, how about a TODO to switch instead of disabling? TBH I'd actually prefer for `formatv` to take `llvm::StringLiteral`, over `.c_str`

Reply:
Yep, makes sense!